src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/TagHelperServiceTestBase.cs

```diff
@@ -17,12 +17,14 @@
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.Completion
 {
-    public abstract class DefaultTagHelperServiceTestBase : LanguageServerTestBase
+    public abstract class TagHelperServiceTestBase : LanguageServerTestBase
     {
+        protected static readonly string ProjectPath = TestProject.GetProjectDirectory(typeof(TagHelperServiceTestBase));
```

Comment:
Does this still need to be on the TagHelperServiceddTestBase or can it be on the SemanticTokenServiceTestBase?